app/src/main/java/com/example/trempelapp/presentation_layer/splash_screen/TrempelSplashActivity.kt

```diff
@@ -42,11 +42,11 @@ class TrempelSplashActivity : BaseActivity() {
         trempelSplashViewModel.isLoggedInLiveData.observe(this, {
 
             if (trempelSplashViewModel.isLoggedInLiveData.value == false) {
```

Comment:
redundant calling of trempelSplashViewModel.isLoggedInLiveData.value. Why are you doing it in the observer? You already now the value, you can either call it or declare a variable here

Reply:
done